apis_core/apis_entities/filters.py

```diff
@@ -69,21 +69,24 @@ class Meta:
         exclude = fields_to_exclude
 
     def __init__(self, *args, **kwargs):
-
         # call super init foremost to create dictionary of filters which will be processed further below
         super().__init__(*args, **kwargs)
         self.entity_class = self.Meta.model
 
         def set_sort_filters(default_filters):
             """
-            Method to read in from the settings file which filters should be enabled / disabled and if there are
-            methods or labels to override the default ones.
-
-            :param default_filter_dict: the default filter dictionary created on filter class instantiation
-                (which comprises filters defined: in GenericListFilter, in specific model ListFilter and their defaults)
-
-            :return: a new dictionary which is a subset of the input dictionary and only contains the filters which
-                are referenced in the settings file (and if there were methods or labels also referenced, using them)
+            Check for existence of "list_filters" setting for an entity class
+            and return an ordered dictionary containing filters for
+            applicable fields plus pre-set filters defined for all entities
+            (in GenericEntityListFilter's Meta class).
+
+            :param default_filters: dictionary of filters created
+                                    on filter class instantiation;
+                                    includes: GenericListFilter, specific
+                                    model ListFilter and their defaults
```

Comment:
I don't fully understand this description, so had to leave it as-is. From what I could see, `default_filters` include a filter for every field defined for an entity...